src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -36,7 +36,8 @@
  */
 public class ParserUtil {
 
-    public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String MESSAGE_INVALID_INDEX =
+            "Index is not a non-zero unsigned integer, or bigger than maximum value.";
```

Comment:
Think should specifically say what is maximum value. A typical user would be confused if you meant the number of candidates or whatever.

Reply:
True. Will make the changes accordingly.